packages/generate-design-tokens/index.js

```diff
@@ -111,7 +125,13 @@ const generateSkinFiles = () => {
 
         const skinSrc = generateSkinSrc(skinName);
         fs.writeFileSync(path.join(SKINS_FOLDER, `${skinName}.tsx`), skinSrc);
+        anyGeneratedSkin = skinName;
     });
+
+    if (anyGeneratedSkin) {
```

Comment:
why using this if statement? As far as I can see,  it will always be true unless `KNOWN_SKINS` is empty